extensions/amp-sidebar/1.0/amp-sidebar.js

```diff
@@ -47,6 +47,21 @@ class AmpSidebar extends PreactBaseElement {
     });
   }
 
+  /** @override */
+  attachedCallback() {
```

Comment:
Since this is a warning and not an assertion, what do you think about running during `init`? I'm not familiar with what our plans are for using classical AMP runtime callbacks, but I had assumed we would try to rely on them as little as possible.

Reply:
I think this callback will always be there since it's just a simple proxy (more or less) of the standard `connectedCallback`. This helps us with the element's reparenting.